test/common/upstream/health_checker_impl_test.cc

```diff
@@ -591,6 +591,7 @@ TEST_F(HttpHealthCheckerImplTest, Degraded) {
   // We start off as healthy, and should go degraded after receiving the degraded health response.
   EXPECT_CALL(*test_sessions_[0]->interval_timer_, enableTimer(_));
   EXPECT_CALL(*test_sessions_[0]->timeout_timer_, disableTimer());
+  EXPECT_CALL(*event_logger_, logDegraded(_, _));
```

Comment:
Should we inspect the logs like in: https://github.com/envoyproxy/envoy/blob/7c14fd1f1a12333712b3e8a279d75cba22a6345d/test/common/upstream/outlier_detection_impl_test.cc#L773